mage_ai/server/api/logs.py

```diff
@@ -109,9 +117,11 @@ def get(self, pipeline_uuid):
             ])).
             join(a, a.id == c.pipeline_run_id).
             join(b, a.pipeline_schedule_id == b.id).
-            filter(b.pipeline_uuid == pipeline_uuid)
+            filter(b.pipeline_uuid == pipeline_uuid).
+            order_by(c.started_at.desc())
```

Comment:
Related to https://github.com/mage-ai/mage-ai/pull/1477/files#r1035480073

Reply:
this works